src/metabase/models/database.clj

```diff
@@ -452,25 +456,20 @@
   [[{:keys [id]}]]
   (t2/select-one Database :name id))
 
-(defmethod serdes/load-xform "Database"
-  [database]
-  (-> database
-      serdes/load-xform-basics
-      (update :creator_id serdes/*import-user*)
-      (assoc :initial_sync_status "complete")))
-
-(defmethod serdes/load-insert! "Database" [_ ingested]
-  (let [m (get-method serdes/load-insert! :default)]
-    (m "Database"
-       (if (:details ingested)
-         ingested
-         (assoc ingested :details {})))))
-
-(defmethod serdes/load-update! "Database" [_ ingested local]
-  (let [m (get-method serdes/load-update! :default)]
-    (m "Database"
-       (update ingested :details #(or % (:details local) {}))
-       local)))
+;;; FIXME: verify with a test that this still stands (it should though)
```

Comment:
did you cover this with tests? I'm interested in how this behaviour is preserved 
https://github.com/metabase/metabase/pull/46147/files#diff-08f05088ade450091841832831efb413eb5204f0cd7eb1f253d5dfb9f7b50dc7L465-L467

Reply:
I did cover it now.